packages/frontend/src/components/Strategies/Crab/CrabTrade.tsx

```diff
@@ -274,12 +274,14 @@ const CrabTrade: React.FC<CrabTradeType> = ({ maxCap, depositedAmount }) => {
           )}
           <div className={classes.tradeContainer}>
             {depositOption === 0 ? (
-              !isQueued ? (
-                <CrabMigration />
+              !currentEthValue.isZero() ? (
+                <Typography variant="body2" color="textSecondary" style={{ marginTop: '20px' }}>
+                  Migrate to V2 coming soon!
```

Comment:
Maybe we should clarify that this is migration for those who didn't migrate during early access?

Something like:
> Didn't migrate from Crab V1 during early access? Migration to V2 coming soon.